docs/getting-started/flashing-firmware/nrf52/ota.mdx

```diff
@@ -45,6 +45,16 @@ OTA firmware updates are available for Android using an older release of the mor
 
 OTA firmware updates are available on iOS & iPadOS using the nRF Device Firmware Update App available through the [Apple App Store](https://apps.apple.com/us/app/nrf-device-firmware-update/id1624454660)
 
+:::caution
```

Comment:
Wondering if instead of putting this in a callout(caution) we should just include this in the step by step instructions

Reply:
If you'd prefer to include the suggested nRF DFU app settings in the step-by-step instructions it still gets the point across. I still think it makes sense to include a caution at the top of the page, above the Android/Apple tabs, regarding the need to re-flash firmware via USB if the OTA flash fails. I think it could prevent a lot of frustration.

I'm not very experienced with GitHub pull requests - Are you able to briefly guide me with the best way to amend my pull request with additional changes to the same file? If you don't have time or don't want to, I completely understand.

Reply:
> Wondering if instead of putting this in a callout(caution) we should just include this in the step by step instructions

Agree with this no need to scare every user